dotCMS/src/main/java/com/dotcms/rest/api/v2/languages/LanguagesResource.java

```diff
@@ -73,6 +84,75 @@ public Response  list(@Context final HttpServletRequest request, @Context final
         return Response.ok(new ResponseEntityView(ImmutableList.copyOf(languages))).build();
     }
 
+    /**
+     * Persists a new {@link Language}
+     *
+     * @param request HttpServletRequest
+     * @param languageForm LanguageForm
+     * @return Response
+     */
+    @POST
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "application/javascript" appears 4 times in this file; the first occurrence is on line 95](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4616952)